llvm/include/llvm/IR/IntrinsicInst.h

```diff
@@ -547,6 +534,64 @@ class DbgLabelInst : public DbgInfoIntrinsic {
   /// @}
 };
 
+/// Wrapper functions for opaque interfacing between DbgVariableIntrinsics and
+/// DPValues.
```

Comment:
IMO this comment is opaque (pun intended), can we specify that these are deliberate overloads to ease writing type-generic code? (Also that they're here temporarily).

Presumably they're in this very-broadly-included header because they're going to be used across the compiler?